docs/manual/meta/hand_tracking.rst

```diff
@@ -0,0 +1,145 @@
+Meta Hand Tracking
+==================
+
+.. note::
+
+    Check out the `Meta Hand Tracking Sample Project <https://github.com/GodotVR/godot_openxr_vendors/tree/master/samples/meta-hand-tracking-sample>`_
+    for a working demo of Meta-specific hand tracking features.
+
+The Godot OpenXR Vendors plugin supports a number of Meta OpenXR extensions related to hand tracking; this tutorial serves as a tour of
+these extensions with an explanation of how they might be used.
+
+To use any of the hand tracking OpenXR extensions, ensure the hand tracking Meta XR feature is enabled in export settings.
```

Comment:
The `xr/openxr/extensions/hand_tracking` project setting must also be enabled. It's enabled by default.